src/translations/index.ts

```diff
@@ -1,51 +1,56 @@
 import {dev} from '$app/environment';
-import type {Config} from 'sveltekit-i18n';
-import i18n from 'sveltekit-i18n';
+import i18n, {type Config} from 'sveltekit-i18n';
 import lang from './lang.json';
 
-export const initialLocale = 'fr'
+interface TranslationLoader {
+    locale: string;
+    key: string;
+    loader: any;
+}
+
+export const initialLocale = 'fr';
+
+const locales: string[] = ['en', 'fr'];
+const components: string[] = ['cookies-disclaimer', 'collection'];
+
+async function loadTranslation(locale: string, key: string): Promise<object> {
+    return (await import(`./${locale}/lib/frontend/components/${key}.json`)).default;
```

Comment:
interesting, so for components we just pass the name of the component and for pages we will pass the name of the page instead, is that so?

Reply:
For components, passing the name alone is sufficient. However, although a similar method is ideal for pages, we must discuss and determine the optimal solution due to their differing paths.

![image](https://github.com/6C-Solutions-4-Small-Businesses/online-shop-template/assets/7751678/7d3df98b-f0c7-48f0-b3b2-116d56b2d58d)